@navikt/core/react/src/popover/Popover.tsx

```diff
@@ -203,22 +203,24 @@ export const Popover = forwardRef<HTMLDivElement, PopoverProps>(
       left: "right",
     }[flPlacement.split("-")[0]];
 
+    const floatingProps = getFloatingProps({
+      ref: floatingRef,
+      style: {
+        position: strategy,
+        top: y ?? 0,
+        left: x ?? 0,
+      },
+    });
+    delete floatingProps.tabIndex;
```

Comment:
Ser ut som du også kan gjøre slik:
```suggestion
        tabIndex: undefined,
      },
    });
```

Reply:
Bra forslag! (Dog tror jeg den skal ned én linje, slik at den havner etter style og ikke inni style-definisjonen.
Men da kan jeg også bare legge det inn i kallet i return, slik det var opprinnelig (= mindre diff).